srcs/incs/HttpPostResponse.hpp

```diff
@@ -18,6 +18,7 @@
 #include <RequestConfig.hpp>
 #include <HttpBaseResponse.hpp>
 #include <HttpErrorResponse.hpp>
+#include <Utils.hpp>
```

Comment:
We have a few functions we don't use anymore in Utils...

Reply:
:+1: I've removed them.